app/controllers/rsvps_controller.rb

```diff
@@ -13,7 +13,8 @@ def create
     respond_to do |format|
       format.html { redirect_to thank_rsvp_url(@event, total_count: total_count) }
     end
-  rescue Constellation::HttpException => e
+  rescue StandardError => e
+    logger.warn(e.backtrace)
```

Comment:
This probably needs to log the exception first, and the backtrace in a readable format (with a `join("\n")`), not sure what the best practice is.

Reply:
This is catching most Ruby exceptions, which means we could be masking real exceptions in this app. As mentioned below, I don’t think we should be raising any exceptions at all in this situation, but in general rescuing `StandardError ` exceptions at this level (rather than e.g. letting Rails’ `rescue_from` handle uncaught exceptions) is not ideal.

Reply:
Even `rescue_from` is not ideal as it's [not recommended by the official guides](http://edgeguides.rubyonrails.org/action_controller_overview.html#rescue-from). At least this should be `Constellation::GraphQLException`.